kubernetes-services/work-scheduler/env-defaults

```diff
@@ -34,7 +34,7 @@ QUEUE_LONG_POLLING_WAIT_TIME_SEC=20
 CMR_MAX_PAGE_SIZE=2000
 
 # SQS queue used to request scheduling of work items for a service
-WORK_ITEM_SCHEDULER_QUEUE_URL=http://localstack:4566/queue/work-item-scheduler-queue
+WORK_ITEM_SCHEDULER_QUEUE_URL=http://localstack:4566/queue/work-item-scheduler-queue.fifo
```

Comment:
Wait - I thought the scheduler queue was _not_ FIFO?

Reply:
oops forgot to revert - good catch